### ISSUE TITLE
Option to center camera on current location on map load

### ISSUE THREAD

**Issue by carterhudson** [googlemaps/android-maps-compose#184]
Thanks for stopping by to let us know something could be better!

---
**PLEASE READ** 

If you have a support contract with Google, please create an issue in the [support console](https://cloud.google.com/support/). This will ensure a timely response. 

Discover additional support services for the Google Maps Platform, including developer communities, technical guidance, and expert support at the Google Maps Platform [support resources page](https://developers.google.com/maps/support/). 

If your bug or feature request is not related to this particular library, please visit the Google Maps Platform [issue trackers](https://developers.google.com/maps/support/#issue_tracker).

Check for answers on StackOverflow with the [google-maps](http://stackoverflow.com/questions/tagged/google-maps) tag.

---

 **Is your feature request related to a problem? Please describe.**
I'd like to be able to configure the map to start with the camera on my current location if something like `isMyLocationEnabled = true`, without having to press a button. It seems like a waste of time to do it all manually when the SDK already has the current location, and can show it if a property is enabled.

 **Describe the solution you'd like**
Perhaps a property in `MapProperties` called `startAtCurrentLocation`.

 **Describe alternatives you've considered**
Requesting the user's current location, then setting it as a camera update on map load.

**Comment by Priyankkjain** [comment]
If you want to start the map at a particular location you can use ```rememberCameraPositionState```'s position argument.

For example 
```
val cameraPositionState = rememberCameraPositionState {
   position = CameraPosition.fromLatLngZoom(currentUserLocaiton, 17f)
}
```
Pass this position state in the GoogleMap composable function and the map will start at that particular location.

**Comment by carterhudson** [comment]
> If you want to start the map at a particular location you can use `rememberCameraPositionState`'s position argument.
> 
> For example
> 
> ```
> val cameraPositionState = rememberCameraPositionState {
>    position = CameraPosition.fromLatLngZoom(currentUserLocaiton, 17f)
> }
> ```
> 
> Pass this position state in the GoogleMap composable function and the map will start at that particular location.

Yes, I'm aware of this. As I stated in my original issue, there shouldn't be a need for me to go through that process if the maps SDK already has my current location.

**Comment by konradzuse** [comment]
I would also find this a useful feature
